### PR TITLE
Update phone number

### PR DESCRIPTION
**Pull request by rpottsoh**
@kotp Since the difficulty level is higher....  Should I add "Abstraction" to "topics" in config.json for this exercise?  Most of the exercises I have utilized an abstraction layer, kind of a habit; I always like to code to the abstraction layer instead of to the class.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -36,26 +36,27 @@
       ]
     },
     {
-      "slug": "bowling",
-      "difficulty": 6,
+      "slug": "bank-account",
+      "difficulty": 2,
       "topics": [
```
Comment (kotp) [inline config.json]: Why is there churn in this file, regarding things that are unrelated to phone number?

File: exercises/saddle-points/uSaddlePointsExample.pas [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+// open-source code library titled Spring4D.  Latest
+// version is available here: https://bitbucket.org/sglienke/spring4d
+// Spring4D is not absolutely necessary, but it does greatly help
+// facilitate a much simpler solution.
+//
+// Spring4D does include a Tuple record type that I have purposely
+// not used in favor of my own, simple, TTuple that I declared in the
+// test runner, in order to remove Spring4D dependency for test execution 
+//========================================================================
 unit uSaddlePoints;
```
Comment (kotp) [inline exercises/saddle-points/uSaddlePointsExample.pas]: Huh?  This is saddle points file, in a PR called Phone Number...

**Comment by kotp** [comment]
Let's get this cleaned up first, and then feel free to add the abstraction in, we can remove it.  This way it is clear what you are indicating.

Right now, it is unclear where you would want it.

**Comment by rpottsoh** [comment]
A note to self.  I have confirmed today, that the changes I made at the end of the day yesterday are working fine in XE5 and Berlin.  Phone number is OK to merge.

**Comment by rpottsoh** [comment]
@kotp Down to 6 commits and 1 file changed.  Squash & Merge?

**Comment by rpottsoh** [comment]
Up to 9. :face_with_head_bandage:
